XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Providers/Controllers/WindowsMixedRealityControllerDataProvider.cs

```diff
@@ -440,10 +440,24 @@ private WindowsMixedRealityMotionController GetController(InteractionSource inte
         private static async void TryRenderControllerModel(InteractionSource interactionSource, WindowsMixedRealityMotionController controller)
         {
 #if WINDOWS_UWP
-            if (!UnityEngine.XR.WSA.HolographicSettings.IsDisplayOpaque) { return; }
+            if (!UnityEngine.XR.WSA.HolographicSettings.IsDisplayOpaque)
```

Comment:
We already check if the hmd is opaque using the camera rig. Let's remove the unity dependencies

Reply:
But does that check in the camera rig affect rendering controller models? Also this is not a change of this PR. I only moved the { return } to the next line.

Reply:
Then I suggest changing this test to use the Camera.isOpaque rather than the Unity dependency.
If the option isn't visible, then fix that.